### PR TITLE
feat: removes xr category

### PR DESCRIPTION
**Pull request by akshbhu**
#### Description of changes
- removes xr plugin
- to de done: warning forXR deprecation

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/src/input-manager.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { getPluginsWithName, getAllPluginNames } from './plugin-manager';
 import { InputVerificationResult } from './domain/input-verification-result';
 import { pathManager, stateManager } from 'amplify-cli-core';
 import { insertAmplifyIgnore } from './extensions/amplify-helpers/git-manager';
-import { runHelp, commandsInfo } from 'amplify-cli-core';
```
Comment (sobolk) [inline packages/amplify-cli/src/input-manager.ts:9]: unused?

**Review by sobolk — COMMENTED** [review]
LGTM. Please run e2e tests.

**Review by evcodes — DISMISSED** [review]
🧹 cleaning up

File: packages/amplify-cli/src/input-manager.ts (hunk 1 above)
Comment (akshbhu) [reply packages/amplify-cli/src/input-manager.ts:9]: yes

**Review by sobolk — DISMISSED** [review]

**Review by sobolk — APPROVED** [review]

**Review by sdstolworthy — APPROVED** [review]

**Review by Amplifiyer — APPROVED** [review]

**Pull request merged**